### ISSUE TITLE
improve lemmatisation

### ISSUE THREAD

**Issue by lukavdplas** [UUDigitalHumanitieslab/tscan#63]
There are some noticable inaccuracies in the output from the frog lemmatiser (such as `*heden` not being lemmatised to `*heid`), perhaps we can improve the lemmatisation.

One option is to add a different lemmatisation service that can be used instead of frog. We should investigate if there is a lemmatiser for Dutch with significantly better results.

Another option is to use some combination of the frog and alpino output for the final lemmatisation. Suggestion from @oktaal 

> Interessant genoeg lijkt de Alpino-parse in dit geval wel "gedweeheid" als lemma te vinden maar die informatie wordt niet gebruikt in T-Scan.
> 
> Wat ik zou kunnen doen is het lemma-attribuut van de Alpino-parse te gebruiken als (1) het lemma van frog hetzelfde is als het woord (dus geen lemmatisering) en (2) het lemma van Alpino wel afwijkt. Als beide een lemma hebben dat afwijkt van het woord dan is Frog leidend. Als Frog correct ziet dat het lemma hetzelfde is als het woord en Alpino er toch wat anders van heeft gemaakt dan introduceert dat dan wel een fout.
> 
> Ik vraag me af in hoeverre hier nieuwe problemen kunnen ontstaan, idealiter zou je dit willen kunnen evalueren. Misschien moet dit een optie worden (lemma-informatie: alleen Frog (nu het geval), alleen Alpino, Frog met Alpino-fallback, Alpino met Frog-fallback).

**Comment by kosloot** [comment]
Some remarks: 
We all know Frog isn't imperfect, but it already knows about 880 different *heden to *heid lemma's, which  isn't that bad.
But the Frog lemmatizer does miss some (not all) versions outside that 880
It isn't that hard to train extra lemma's into frogs datafiles. All you need is a list of 
``word <tab> lemma <tab> POStag`` 
cases.
In that way you could improve rather quickly imho,
Background: The lemmatizer is trained on sentences from CGN with additions from an extra list of know lemma's
which can easily be expanded. With other missing lemmas too.
Training can be done using ``froggen``. Not a very difficult task, once used to it :P 
If you have a list in the right format, I am willing to do this task, IFF we may use this data to add to the Frog project as a whole.
(see also [the froggen manual](https://frognlp.readthedocs.io/en/latest/advanced.html)

**Comment by lukavdplas** [comment]
Thanks! I'm not very familliar with frog myself, so I did not know that it could be retrained. This may be the best way forward for us, what do you think, @oktaal ?

**Comment by oktaal** [comment]
Definitely. Thank you! We are going to collect a new list for Frog. You will see it some time in the future @kosloot

**Comment by kosloot** [comment]
looking forward.In the meantime I improved on froggen a bit, to make everybody's life mor comfortable.

NB: the POS tags must be from the CGN set.
